examples/platform/nxp/common/device_callbacks/source/CommonDeviceCallbacks.cpp

```diff
@@ -173,14 +173,6 @@ void chip::NXP::App::CommonDeviceCallbacks::OnSessionEstablished(chip::DeviceLay
 #if CHIP_ENABLE_OPENTHREAD
 void chip::NXP::App::CommonDeviceCallbacks::OnComissioningComplete(const chip::DeviceLayer::ChipDeviceEvent * event)
 {
```

Comment:
@doru91 - this seemed reasonable to me as the enable flag in ember is not actually used.

Please confirm/verify.

Reply:
Hi @andy31415, yes you can remove this code. The original intent was to disable the WIFI commissioning interface on a multi interface device when the device was first commissioned over Thread. In this case it should remain a Matter over Thread device and could not transition to an OTBR role by enabling WIFI. OTBR could be achieved only on a factory reset device by first commissioning over WIFI and then using the TBRM cluster to set up the Thread interface.  I agree the flag is not used so you can remove the code and I will consider another approach.